packages/core/src/generators/interface.ts

```diff
@@ -53,6 +53,7 @@ export const generateInterface = ({
   } else {
     model += `export type ${name} = ${scalar.value};\n`;
   }
+  model += `export function create${name}(): ${name} ${scalar.factoryMethodValue}\n`;
```

Comment:
Factory methods are just noise for people who don't need them, so I'd like to be able to choose whether to generate them with a property.
For example, `generateFactoryMethod: boolean`.

And by making it a property, people who want to use it will be aware that it will be generated. Therefore, you will be able to notice it even if you separate the file like `.factory.ts`, right?